src/index.html

```diff
@@ -7,12 +7,64 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="imac"
+      />
+
+      <div class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__code">Product code: 195434</div>
+
+      <div class="stars">
+        <div class="stars__block">
+          <div class="stars__star stars__star--1"></div>
+          <div class="stars__star stars__star--2"></div>
+          <div class="stars__star stars__star--3"></div>
+          <div class="stars__star stars__star--4"></div>
+          <div class="stars__star stars__star--5"></div>
+        </div>
```

Comment:
Fix class and add the modifier class, because you need use this block from the previous task
```suggestion
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
```

Reply:
+++